openshift-kube-apiserver/admission/autoscaling/managementcpusoverride/admission.go

```diff
@@ -180,6 +180,11 @@ func (a *managementCPUsOverride) Admit(ctx context.Context, attr admission.Attri
 		return admission.NewForbidden(attr, err) // can happen due to informer latency
 	}
 
+	// we can not decide the cluster type without Status.ControlPlaneTopology and Status.InfrastructureTopology
+	if clusterInfra.Status.ControlPlaneTopology == "" || clusterInfra.Status.InfrastructureTopology == "" {
+		return admission.NewForbidden(attr, fmt.Errorf("%s infrastructure resource %q has empty Status.ControlPlaneTopology or Status.InfrastructureTopology", PluginName, infraClusterName))
```

Comment:
lower-case field names

Reply:
the object name is redundant. There is only one.

Reply:
done